loleaflet/src/layer/tile/CanvasTileLayer.js

```diff
@@ -125,8 +113,8 @@ L.CanvasTilePainter = L.Class.extend({
 	_syncTileContainerSize: function () {
 		var tileContainer = this._layer._container;
 		if (tileContainer) {
-			tileContainer.style.width = this._width + 'px';
```

Comment:
This, together with the _setCanvasSize (and whatever related) would be worth a separate commit, it is very non-obvious for code reading I fear...